Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -359,6 +359,7 @@
                                                'keep HBHEDataFramesSorted_hcalDigis_*_*',
                                                'keep HFDataFramesSorted_hcalDigis_*_*',
                                                'keep HODataFramesSorted_hcalDigis_*_*',
+                                               'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_*_*',
```

Comment:
As far as I know, only HF will have QIE10. HB and HE will have QIE11 for their respective upgrades. The full name for, e.g., the output of simHcalUnsuppressedDigis ends up being:
```
QIE10DataFrameHcalDataFrameContainer_simHcalUnsuppressedDigis_HFQIE10DigiCollection_HLT
```

So the HF name does show up there.